### ISSUE TITLE
New user accounts must manually re-select team

### ISSUE THREAD

**Issue by jakebanderson321** [andersonopt/app-user-docs#176]
### Subject of the issue
When I create a new user account and then add them to a team, I have to go back into their account and re-select the team from the user profile (even though it's already selected) and do a refresh to get the team's data to show up. 

VERY minimal issue but just an extra step for account creation and requires a refresh.

New user account after being added to a team but before re-selecting it from user profile

![image](https://user-images.githubusercontent.com/13791320/55892292-ae9adb00-5b7b-11e9-846e-806c7ed0232e.png)

Shows current team is correct but the data is still not showing. Have to reselect the team and refresh to display properly

![image](https://user-images.githubusercontent.com/13791320/55892363-d8540200-5b7b-11e9-9a81-b046b2d209ff.png)

**Comment by eanderson4** [comment]
Probably ideal behavior is if you end up on the dashboard with no team selected, it should look for existing teams and if you are on any, it should select it and refresh the page.

Would that do the trick?

**Comment by jakebanderson321** [comment]
Yep, that would do the trick. Even automatically defaulting to the first team a user is added to would be fine.

What's weird to me is that it shows a "Current Team" selected (see screenshot above) like it's attempting to do that. It just doesn't seem to "trigger" until re-selecting what's already showing as selected and then refreshing.
